exporters/otlp/CMakeLists.txt

```diff
@@ -4,9 +4,11 @@ add_library(opentelemetry_exporter_otprotocol src/recordable.cc)
 target_link_libraries(opentelemetry_exporter_otprotocol
                       $<TARGET_OBJECTS:opentelemetry_proto>)
 
-add_executable(recordable_test test/recordable_test.cc)
-target_link_libraries(
-  recordable_test ${GTEST_BOTH_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT}
-  opentelemetry_exporter_otprotocol protobuf::libprotobuf)
-gtest_add_tests(TARGET recordable_test TEST_PREFIX exporter. TEST_LIST
+if(BUILD_TESTING)
+  add_executable(recordable_test test/recordable_test.cc)
+  target_link_libraries(
+    recordable_test ${GTEST_BOTH_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT}
+    opentelemetry_exporter_otprotocol protobuf::libprotobuf)
+  gtest_add_tests(TARGET recordable_test TEST_PREFIX exporter. TEST_LIST
                 recordable_test)
```

Comment:
I think if you indent this line too, the `format` check will pass.

Running `./ci/run_docker.sh ./ci/do_ci.sh format` in your repository helps you to verify that locally.

Reply:
Thanks, will try that tomorrow, since it seems to take quite a while